src/step_definitions/elements.js

```diff
@@ -20,58 +20,58 @@ defineSupportCode(function ({ When, Then }) {
   });
 
   When(/^I click the "([^"]*)" element$/, function (elementName) {
-    const self = this;
-
-    return self.currentPage.scrollIntoElement(elementName)
-      .then(function () {
-        return self.currentPage.click(elementName)
-          .then(
-            null,
-            function () {
-              console.warn('Warning! Element was not clickable. We need to scroll it down.');
-              return browser.executeScript('window.scrollBy(0,50);')
-                .then(function () {
-                  return self.currentPage.click(elementName)
-                    .then(
-                      null,
-                      function () {
-                        return Promise.reject(`Error, after scrolling the element "${elementName}" is still not clickable.`);
-                      }
-                    );
-                });
-            }
-          );
+    return this.currentPage.waitForVisibilityOf(elementName)
+      .then(() => {
+        this.currentPage.scrollIntoElement(elementName);
```

Comment:
This may not work as expected - the arrow function is returning null. Same below